.travis.yml

```diff
@@ -1,14 +0,0 @@
-sudo: false
-language: node_js
-node_js:
-  - '0.12'
-deploy:
-  provider: npm
-  email: d2ltravisdeploy@d2l.com
-  api_key:
-    # d2l-travis-deploy: f679......733c
```

Comment:
I'll remove this NPM token.

Reply:
Also https://github.com/Brightspace/d2l-fetch-auth/pull/31/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L27 (in case you hadn't already seen that one)